src/js/containers/sharedContainers/LocationListContainer.jsx

```diff
@@ -96,15 +55,13 @@ class LocationListContainer extends React.Component {
                 formatter={this.dataFormatter}
                 handleTextInput={this.handleTextInput}
                 onSelect={this.props.selectLocation}
-                placeHolder="State, City, County, Zip or District" />
+                placeHolder="State, City, County, ZIP, or District"
+                ref={(input) => {
+                    this.locationList = input;
+                }} />
         );
     }
 
 }
 
```

Comment:
Why has this container been disconnected from Redux? If a container is no longer required, delete the entire file and just use the child component. However, remember that components should be _dumb_, in other words they do not directly make API calls, they only perform logic related to display and layout.